tron/bin/action_runner.py

```diff
@@ -103,12 +103,19 @@ def parse_args():
     return parser.parse_args()
 
 
-def run_command(command):
+def run_command(command, run_id):
+    namespace, job, run_num, action = run_id.split('.')
     return subprocess.Popen(
         command,
         shell=True,
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
+        env=dict(
```

Comment:
I'm afraid that this is a breaking change compared to the previous which inherited the original env, including PATH, etc.

Can you make that not the case?

Reply:
Done.

Reply:
```
drmorr@dev54-uswest1adevc ~tron > tronview --server=http://dev54-uswest1adevc.dev.yelpcorp.com:8089 MASTER.testjob0.1.zeroth
Action Run          : MASTER.testjob0.1.zeroth
State               : succeeded
Node                : drmorr@localhost
Command             : env
Bare command        : env
Start time          : 2019-10-11 15:44:00
End time            : 2019-10-11 15:44:00
Final exit status   : 0
Retry exit statuses : []
Waits for triggers  :
Publishes triggers  : minutely.2019-10-11T15:44
Requirements:

Stdout:
XDG_RUNTIME_DIR=/run/user/4048
LANG=en_US.UTF-8
TRON_ACTION=zeroth
TRON_JOB_NAME=testjob0
SHELL=/bin/zsh
GIT_COMMITTER_EMAIL=drmorr@yelp.com
PWD=/nail/home/drmorr
SSH_CONNECTION=127.0.0.1 39642 127.0.0.1 22
EDITOR=vim
TRON_JOB_NAMESPACE=MASTER

Stderr:

```